mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -188,7 +188,6 @@ private boolean isDataSaveNecessary(EditText input) {
     public void onUserDetailsFocusChanged(EditText input, boolean isFocused) {
         if (!isDataSaveNecessary((input))) {
             if (isFocused) {
```

Comment:
change this to `if(!isFocused) clear the text`

Reply:
Done sir